app/errors.py

```diff
@@ -32,3 +34,18 @@ def limit_exceeded(e):
     def internal_server_error(e):
         current_app.logger.exception(e)
         return jsonify(error="Internal error"), 500
+
+    @blueprint.app_errorhandler(NoResultFound)
+    def no_result_found(e):
```

Comment:
This is good. In the future we could add to the error message by examining exception to determine exactly what object they were trying to retrieve.